notebooks/notebooks.yaml

```diff
@@ -0,0 +1,14 @@
+stages:
+  results-mind-val:
+    cmd: jupytext --to notebook --execute mind-val.md
+    deps:
+      - notebooks
```

Comment:
The dependencies need to be:

- the markdown file
- the input metrics file

All paths need to be relative to this directory, not project root.